### DISCUSSION TITLE
Как присвоить чтобы график начинался с нуля?

### DISCUSSION THREAD

**Discussion by Re-Requeste** [totumonline/totum-ru-issues-and-discussions#494] [Как это сделать?] [answered]
Добрый день, как присвоить графику чтобы он начинался с 0. 
Сейчас у меня мин.значение которое собирается в dataset через SelectRowList, и мин. значение 60. И график начинается с 60.
Выглядит так: 
![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/137899742/5d6c433e-3565-493c-a66c-99ef5d09434c)

В самом JS он начинается с 0, какие факторы влияют на это? 
![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/137899742/8157614b-4cfa-4176-ae5f-3d2b2084cf3f)

**Comment by Re-Requeste** [comment, marked as answer]
Нашел такое решение , добавить :
`  "scales": {
    "xAxes": [
      [
        {
          "stacked": true
        }
      ]
    ],
    "yAxes": [
      {
        "stacked": true
      }
    ]
  },`

**Reply by ttm-alexeypolunin** [reply]
Спасибо, что покопались у них в доках, тк я знаю еще людей, которым эти настройки осей были нужны!

Нам надо еще будет обновить эти графики на последнюю версию тк тотум использует версию  3.5. Проблема в том, что у них настройки обратно несовместимы. Рабочая гипотеза — переключатель меду версиями графиков.
